src/components/StoryCreationComponent.js

```diff
@@ -12,45 +12,29 @@ import { Col, Row } from 'reactstrap';
 
 import '../styles/AttributeDeselector.css';
 
+//React quill
+import * as ReactQuill from 'react-quill'; // Typescript
+import 'react-quill/dist/quill.snow.css'; // ES6
+
 class StoryCreationComponent extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
       isLoaded: false,
-      selectedAttributes: [],
       authorField: '',
       titleField: '',
       tagsInputValue: '',
       tagsField: [],
       storyField: '',
+      storyTextOnly: '',
       storyMaxLength: 1000
     };
-    // Set initial state of each collection to false
+    // this.setState = this.setState.bind(this)
   }
 
   componentDidMount() {
     this.setState({ isLoaded: true });
```

Comment:
We're not loading anything when the component mounts right? You can remove `isLoaded` from state

Reply:
isLoaded is currently used as a toggle between showing/hiding this component. A feature request was made to make it so that we can easily disable adding stories. Can change it , but isLoaded seemed to fit.

Reply:
Ah okay, then I think it can stay, but should prolly be named sth else instead of `isLoaded`, `featureDisabled` seems to be more fitting

Reply:
Just kept it for consistency, but can rename easily.